ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -155,38 +161,85 @@ void CurlNetworking::processNetworkRequest(CURLM *curlMultiHandle) {
 
 }
 
-bool CurlNetworking::preparePostRequest(shared_ptr<CurlRequest> curlRequest, folly::dynamic data ) {
-  const char *dataPtr = NULL;
+size_t CurlNetworking::read_callback(void* ptr, size_t size, size_t nmemb, void* userdata) {
```

Comment:
why the api name is read_callback instead readCallback ? any reason for deviating it from standard.